client/www/pages/docs/permissions.md

```diff
@@ -291,3 +291,85 @@ delete to only succeed on todos associated with a specific user email.
 ```
 
 See [managing users](/docs/users) to learn more about that.
+
+### ruleParams
```

Comment:
Took a quick stab at editing here: 

https://gist.github.com/stopachka/3d902de5abdb558e9bc01a381a3b83e5

Main changes: 
1. Added syntax highlighting. 
2. Changed the structure of some sentences.